CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v0.38.0 (unreleased)
+
+- New "Miscellaneous" layer: Exclusive Economic Zone
+- Support adding layers from data contained in a private repository.
```

Comment:
Maybe add parenthetical:

```
 - Support adding layers from data contained in a private repository (just a special directory exposed as a docker volume mount for now).
```

"repository" comes with context that may confuse people initially.